src/app/modules/vault-filter/organization-vault-filter.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Input } from "@angular/core";
+
+import { Organization } from "jslib-common/models/domain/organization";
+
+import { VaultFilterComponent } from "./vault-filter.component";
+
+@Component({
+  selector: "app-organization-vault-filter",
+  templateUrl: "vault-filter.component.html",
+})
+export class OrganizationVaultFilterComponent extends VaultFilterComponent {
+  @Input() hideOrganizations = true;
+  @Input() hideFavorites = true;
+  @Input() hideFolders = true;
```

Comment:
We should remove the `@Input()` decorator on these declarations, as we never expect to use them as inputs in this subclass. They're effectively just constants used in the template.

```suggestion
  hideOrganizations = true;
  hideFavorites = true;
  hideFolders = true;
```